crates/cli-support/src/js/mod.rs

```diff
@@ -2195,6 +2195,20 @@ impl ExportedClass {
     }
 }
 
+/// Returns a sorted iterator over a hash mpa, sorted based on key.
+///
+/// The intention of this API is to be used whenever the iteration order of a
+/// `HashMap` might affect the generated JS bindings. We want to ensure that the
+/// generated output is deterministic and we do so by ensuring that iteration of
+/// hash maps is consistently sorted.
+fn sorted_iter<K, V>(map: &HashMap<K, V>) -> impl Iterator<Item = (&K, &V)>
```

Comment:
Is this better than using `BTreeMap` instead of `HashMap`?

Reply:
Eh I think it's sort of taste. I personally like to use `HashMap` as much as possible until proven otherwise, and then when it affects output either use a temporary `BTreeMap` or sort it like this somehow. It really could go either way though and I don't have a preference one way or another.